modules/utils/docker/entrypoint.sh

```diff
@@ -416,7 +425,15 @@ EOF
     echo "Mesh Point + AP done."
   else
     mesh_service
+    # Start only mesh service with out bridge for network boot as ethernet port will be dedicated
+    # to network boot.
+    if [ -f /root/build.info ]; then
+            if grep -Fxq "TCDIST_PLATFORM=\"cm4io_nfs\"" /root/build.info ; then
+                    mesh_service
+            fi
+    else
     /bin/bash "$BRIDGE_SETTINGS"
+    fi
```

Comment:
Why we are starting mesh_service two times in one specific case?

Reply:
ahh it was typo, fixed. But probably will investigate why ethX is not transparent in NFS boot. Then we can finalize on this change.